tox.ini

```diff
@@ -57,8 +57,7 @@ commands =
 [testenv:static]
 description = Run static type checker
 deps =
-    pyright==1.1.264
-    -r{toxinidir}/requirements.txt
+    -r{toxinidir}/requirements-dev.txt
```

Comment:
Can we also do this for the `[testenv:fmt]` and `[testenv:lint]` sections above, and their dependencies like `isort` and `autopep8`? Would be good if all dev deps were in one place.

Reply:
So, I had initially left these deps as it is because I wanted to keep the dev requirements list minimal. However, now when I tried combining these deps under one `requirements-dev.txt`, I notice that `autopep8` & `flake-*` dep versions are incompatible.

Let me check if I can arrive at a version that can works across all these deps.